KBH/6Day/이상한 문자 만들기.js

```diff
@@ -0,0 +1,19 @@
+function solution(s) {
+    let answer = [];
+    const character = s.split(" ");
+    
+    for (let i = 0; i < character.length; i++) {
+        let arrayPushed = [];
+        for (let j = 0; j < character[i].length; j++) {
+            if ((j % 2) === 0 || j === 0) {
+                arrayPushed.push(character[i][j].toUpperCase());
+            } else {
+                arrayPushed.push(character[i][j].toLowerCase());
+            }
+        }
+        
+        answer.push(arrayPushed.join(""));
+    }
+    
+    return answer.join(" ");
+}
```

Comment:
```suggestion
function solution(s) {
    const answer = []; // answer 은 재할당이 안됩니다. 따라서 const로 정의
    const characters = s.split(" "); // 배열은 복수형태로 작성하는게 일반적입니다.
    
    for (let i = 0; i < characters.length; i++) {
        const arrayPushed = []; // 재할당이 안됩니다. 따라서 const로 정의
        // 맥락 분리를 위한 줄바꿈
        for (let j = 0; j < characters[i].length; j++) {
            if ((j % 2) === 0 || j === 0) {
                arrayPushed.push(characters[i][j].toUpperCase());
            } else {
                arrayPushed.push(characters[i][j].toLowerCase());
            }
        }
        
        answer.push(arrayPushed.join(""));
    }
    
    return answer.join(" ");
}
```
@KimBYH  아래 코멘트 읽어보셨을까요?
https://github.com/modern-agile-team/9term-assignment/pull/38#discussion_r1876351432

Reply:
코드 컨벤션을 지킬려고 노력하고 있는데 아직도 많이 부족한거 같습니다. 몇번 더 검토하고 pr올리도록하겠습니다!

Reply:
코드 컨벤션을 지키도록 더 노력하겠습니다!

Reply:
@KimBYH 갑작스레 여러 피드백이 들어오고, 특히 세심하게 코드를 들여다보는 습관은 한번에 체화되는게 쉽지 않다고 생각합니다ㅎㅎ 초반에는 계속 피드백을 받겠지만 체화되게끔 꾸준히 연습해주세요! 😄